### PR TITLE
updates to user personal info

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**
just removing static tag and setting up the FE to receive new data

Dual Merge: [BE](https://github.com/MetaPhase-Consulting/State-TalentMAP-API/pull/409) & [MOCK](https://github.com/MetaPhase-Consulting/mock-fsbid/pull/122)

### CODE REVIEW

File: src/actions/userProfile.js [hunk 1]
```diff
@@ -66,7 +66,6 @@ export function userProfileFetchData(bypass, cb) {
     /**
      * create functions to fetch user's profile and permissions
      */
-    // TODO: update to use Person endpoint. Grab the perdet, user grade, and user skills, if any.
```
Comment (elizabeth-jimenez) [inline src/actions/userProfile.js:69]: we are already grabbing the grade and skills from the Person EP.

Comment (elizabeth-jimenez) [reply src/actions/userProfile.js:69]: Pulling `perdet` from `/userInfo`, do we want to switch to grabbing it from `/Person`?

**Comment by mjoyce91** [comment]
@elizabeth-jimenez should we expect to see this not only on the user's dashboard, but their public profile via CDO portfolio as well?

**Comment by elizabeth-jimenez** [comment]
> @elizabeth-jimenez should we expect to see this not only on the user's dashboard, but their public profile via CDO portfolio as well?

Ah! I knew I had to address both but it slipped my mind; thanks for the reminder! I'll address this today :) I would assume they want the information on non-public and public profile, but i'll make sure to bring it up in the demo, to make sure that's the case.

File: src/Components/ProfileDashboard/UserProfile/UserProfileContactInformation/UserProfileContactInformation.jsx [hunk 2]
```diff
@@ -11,18 +11,14 @@ const UserProfileContactInformation = ({ userProfile }) => (
     <div className="section-padded-inner-container">
       <SectionTitle small title="Contact Information" icon="list-alt" />
       <InformationDataPoint title="Email address" content={get(userProfile, 'user.email') || NO_EMAIL} />
-      <StaticDevContent>
-        <InformationDataPoint title="Office number" content="+301-779-0379 ext. 3" />
-      </StaticDevContent>
+      <InformationDataPoint title="Office number" content={get(userProfile, 'user_info.office_phone') || NO_OFFICE_PHONE} />
```
Comment (mjoyce91) [inline src/Components/ProfileDashboard/UserProfile/UserProfileContactInformation/UserProfileContactInformation.jsx:14]: Wonder if we want to use the `tel` tag to make it clickable/callable:

`<a href={`tel:${user_info.office_phone}`}>...`

Comment (elizabeth-jimenez) [reply src/Components/ProfileDashboard/UserProfile/UserProfileContactInformation/UserProfileContactInformation.jsx:14]: yes!

File: src/Components/ProfileDashboard/UserProfile/UserProfileContactInformation/UserProfileContactInformation.jsx [hunk 3]
```diff
@@ -10,19 +10,15 @@ const UserProfileContactInformation = ({ userProfile }) => (
   <div className="current-user-section-container">
     <div className="section-padded-inner-container">
       <SectionTitle small title="Contact Information" icon="list-alt" />
-      <InformationDataPoint title="Email address" content={get(userProfile, 'user.email') || NO_EMAIL} />
+      <InformationDataPoint title="Email address" content={<a href={`mailto:${get(userProfile, 'user.email')}`} >{get(userProfile, 'user.email') || NO_EMAIL}</a>} />
+      <InformationDataPoint title="Office number" content={<a href={`tel:${get(userProfile, 'user_info.office_phone')}`} rel="nofollow">{get(userProfile, 'user_info.office_phone') || NO_OFFICE_PHONE}</a>} />
```
Comment (mjoyce91) [inline src/Components/ProfileDashboard/UserProfile/UserProfileContactInformation/UserProfileContactInformation.jsx:14]: 🤩

File: src/Components/ProfileDashboard/UserProfile/UserProfileContactInformation/UserProfileContactInformation.jsx [hunk 4]
```diff
@@ -10,19 +10,15 @@ const UserProfileContactInformation = ({ userProfile }) => (
...
-      </StaticDevContent>
-      <StaticDevContent>
-        <InformationDataPoint title="Personal contact number" content="+240-331-7189" />
-      </StaticDevContent>
-      <StaticDevContent>
-        <InformationDataPoint
-          title="Post/Office address"
-          content="1234 Washington St. NW, Washington, DC 20009"
-        />
+        <InformationDataPoint title="Personal contact number" content={<a href="tel:+240-331-7189" rel="nofollow">+240-331-7189</a>} />
```
Comment (mjoyce91) [inline src/Components/ProfileDashboard/UserProfile/UserProfileContactInformation/UserProfileContactInformation.jsx:16]: 💓

File: src/Components/ProfileDashboard/UserProfile/UserProfileContactInformation/UserProfileContactInformation.jsx (hunk 3 above)
Comment (elizabeth-jimenez) [reply src/Components/ProfileDashboard/UserProfile/UserProfileContactInformation/UserProfileContactInformation.jsx:14]: 🤫  Just don't look at it when they have no info 🤫  hehe, have to address that

Comment (mjoyce91) [reply src/Components/ProfileDashboard/UserProfile/UserProfileContactInformation/UserProfileContactInformation.jsx:14]: Ohh well, let's address that:

```jsx
...
content={{get(userProfile, 'user_info.office_phone')} ? <a href={`tel:${get(userProfile, 'user_info.office_phone')}`} rel="nofollow">{get(userProfile, 'user_info.office_phone')}</a> : NO_OFFICE_PHONE}
...
```

should do it

File: src/Components/ProfileDashboard/UserProfile/UserProfileContactInformation/UserProfileContactInformation.jsx (hunk 4 above)
Comment (mjoyce91) [reply src/Components/ProfileDashboard/UserProfile/UserProfileContactInformation/UserProfileContactInformation.jsx:16]: If this is a fake phone number, don't we need it in StaticDevContent?

Comment (elizabeth-jimenez) [reply src/Components/ProfileDashboard/UserProfile/UserProfileContactInformation/UserProfileContactInformation.jsx:16]: I thought it still was :(

**Comment by elizabeth-jimenez** [comment]
Last thing to do on here is confirm the public profile info is properly coming through from the API, if not, should be updated.

File: src/Components/ProfileDashboard/UserProfile/UserProfileContactInformation/UserProfileContactInformation.jsx [hunk 5]
```diff
@@ -10,19 +10,15 @@ const UserProfileContactInformation = ({ userProfile }) => (
   <div className="current-user-section-container">
     <div className="section-padded-inner-container">
       <SectionTitle small title="Contact Information" icon="list-alt" />
-      <InformationDataPoint title="Email address" content={get(userProfile, 'user.email') || NO_EMAIL} />
+      <InformationDataPoint title="Office number" content={get(userProfile, 'user_info.office_phone') ? <a href={`mailto:${get(userProfile, 'user_info.email')}`} rel="nofollow">{get(userProfile, 'user_info.email')}</a> : NO_EMAIL} />
+      <InformationDataPoint title="Office number" content={get(userProfile, 'user_info.office_phone') ? <a href={`tel:${get(userProfile, 'user_info.office_phone')}`} rel="nofollow">{get(userProfile, 'user_info.office_phone')}</a> : NO_OFFICE_PHONE} />
```
Comment (mjoyce91) [inline src/Components/ProfileDashboard/UserProfile/UserProfileContactInformation/UserProfileContactInformation.jsx:14]: You have `title="Office number"` twice. One of them should be "email"

File: src/Components/ProfileDashboard/UserProfile/UserProfileContactInformation/UserProfileContactInformation.jsx (hunk 4 above)
Comment (mjoyce91) [reply src/Components/ProfileDashboard/UserProfile/UserProfileContactInformation/UserProfileContactInformation.jsx:16]: Nvm maybe I missed something

File: src/Components/ProfileDashboard/UserProfile/UserProfileContactInformation/UserProfileContactInformation.jsx (hunk 5 above)
Comment (elizabeth-jimenez) [reply src/Components/ProfileDashboard/UserProfile/UserProfileContactInformation/UserProfileContactInformation.jsx:14]: Ah, good catch!

**Comment by elizabeth-jimenez** [comment]
Some employees have the same addresses (ex: Elizabeth and Jenny) not bc the mapping is wrong, but bc there are duplicate addresses in the seed file.

**Review by mjoyce91 — APPROVED** [review]

**Pull request merged**